File: examples/py/coinone-fetch-tickers.py
# -*- coding: utf-8 -*-

import os
import sys
from pprint import pprint

root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
sys.path.append(root + '/python')

import ccxt  # noqa: E402

exchange = ccxt.coinone({
    'enableRateLimit': True,
    # 'verbose': True,  # uncomment for verbose output
})

# fetch all
tickers = exchange.fetch_tickers()
for symbol, ticker in tickers.items():
    print(ticker)

print("\n")

# fetch one by one
markets = exchange.load_markets()
for symbol in markets.keys():
    ticker = exchange.fetch_ticker(symbol)
    print(ticker)

File: examples/php/coinone-fetch-tickers.php
<?php

$root = dirname (dirname (dirname (__FILE__)));

include $root . '/ccxt.php';

$exchange = new \ccxt\coinone (array (
    'enableRateLimit' => true,
    // 'verbose' => true, // uncomment for verbose output
));

// fetch all
$tickers = $exchange->fetch_tickers();
var_dump ($tickers);

echo "\n";

// fetch one by one
$markets = $exchange->load_markets();
foreach ($markets as $symbol => $m) {
    var_dump($exchange->fetch_ticker($symbol));
}

?>
